roborazzi-compose/src/main/java/com/github/takahirom/roborazzi/RoborazziCompose.kt

```diff
@@ -118,8 +131,10 @@ private fun ActivityScenario<out ComponentActivity>.captureRoboImage(
         val composeView = activity.window.decorView
           .findViewById<ViewGroup>(android.R.id.content)
           .getChildAt(0) as ComposeView
+
         @SuppressLint("VisibleForTests")
         val viewRootForTest = composeView.getChildAt(0) as ViewRootForTest
+        doBeforeCapture()
```

Comment:
@takahirom 
I'm open to other solutions, but it does have to happen at this point. That was the reason why it was not working previously: the `advanceTimeBy()` was being before the Composable was set as content in the Activity